src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/MessageDrivenSubscribeTerminator.cs

```diff
@@ -28,7 +28,7 @@ protected override Task Terminate(ISubscribeContext context)
             var publisherAddresses = subscriptionRouter.GetAddressesForEventType(eventType);
             if (publisherAddresses.Count == 0)
             {
-                throw new Exception($"No publisher address could be found for message type {eventType}. Ensure the configured publisher endpoint has at least one known instance.");
+                throw new Exception($"No publisher address could be found for message type {eventType}. Ensure that a publisher has been configured for the event type and that the configured publisher endpoint has at least one known instance.");
```

Comment:
`{eventType}` should be in `'`. Wondering why there is no test failing as the acceptance test contains the ``'?

Reply:
fixed

Reply:
ah, v6 doesn't have the inmem persistence tests. I changed the tests locally to use MSMQ as the transport and this test indeed failed.